map/bookmark_manager.cpp

```diff
@@ -2773,21 +2790,21 @@ void BookmarkManager::PrepareFileForSharing(kml::GroupIdCollection && categories
 
   if (m_testModeEnabled)
   {
-    handler(GetFileForSharing(std::move(collection)));
+    handler(GetFileForSharing(std::move(collection), kmlFileType));
   }
   else
   {
     GetPlatform().RunTask(Platform::Thread::File,
-                          [collection = std::move(collection), handler = std::move(handler)]() mutable
-                          { handler(GetFileForSharing(std::move(collection))); });
+                          [collection = std::move(collection), handler = std::move(handler), kmlFileType = std::move(kmlFileType)]() mutable
```

Comment:
kmlFileType is a simple POD type, it can be efficiently copied, the move is not needed.

Just leave `kmlFileType` there.

Reply:
Fixed